packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -201,6 +206,9 @@ const actionsShape = PropTypes.arrayOf(
   PropTypes.shape({
     // groupedActions: the function will be called with an array of the selected items in parameters
     // individualActions: the function will be called with the related item in parameters
+
+    // advanced actions are collapsed in a dropdown
+    advanced: PropTypes.bool,
```

Comment:
It can be a non-advanced action, maybe `dropdowAction` or something like that.

Reply:
The functionality is about collapsing advanced actions, so the dropdown will not contain a non-advanced action.

Reply:
IMHO, it should be for all actions to regroup them.

Reply:
Why should we collapse primary actions?

It's the same logic that we have done to the backup settings. We display primary settings and we collapse advanced settings.